prog/prog.go

```diff
@@ -11,6 +11,7 @@ type Prog struct {
 	Target   *Target
 	Calls    []*Call
 	Comments []string
+	Files    []string
```

Comment:
prog already contains target and files come from target, so this looks unnecessary
the code can just use p.Target.Files where necessary
I suspect there are already some places, where you forget to init p.Files.